docs/settingUp.md

```diff
@@ -24,7 +24,7 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
    ```sh
    git remote add upstream https://github.com/TEAMMATES/teammates.git
    ```
-  
```

Comment:
It's the whitespace. Usually, it is your editor that makes this kind of changes for you.